index.js

```diff
@@ -153,6 +153,30 @@ function renderCodeBlocks($, displayCodeBlockSelectors) {
       );
     };
 
+    const addLineNumbersIfNecessary = () => {
+      if (className && className.indexOf("numberLines") >= 0) {
+        // Add line numbers
+        const $pre = $block.parent();
+        $pre.addClass("line-numbers");
+        $pre.addClass("language-line-numbers"); // language-* in order to apply PrismJS line numbers
+        const start = parseInt($block.attr("startfrom") || "1") || 1;
```

Comment:
@yiyi-wang-rv `startFrom` can start at any integer, isn't it the point of having such an attribute? We can argue that line numbers should be positive integers but this would make the grammar more complicated for little benefit. 

If you are discussing this because of this line, it can stay as is, because having no `startfrom` is different from having startfrom set to `0`.

Reply:
Yes my code handles both `0` and `1`. If you write `startFrom="0"` then it will still show `1` as the initial line. So I think we are good to go

Reply:
@fiedlr ^^^

Reply:
@yiyi-wang-rv I can't give you approval because I'm not on your team.

But personally I would avoid such JS tricks and if `start` is `NaN`, then return and show an error in the console or something. The grammar seems also to support negative integers by the way because the list of tokens contains `-`. Does your code support it?

Reply:
You also set `data-startfrom` to `$block.attr("startfrom")`, regardless of `start`.